### PR TITLE
Update projects

### PR DESCRIPTION
**Pull request by elevenlabs-laura**
- add frames to all images
- add section on auto-regenerate
- add screenshots showing how to enable auto-regenerate

### CODE REVIEW

File: product/projects/overview.mdx [hunk 1]
```diff
@@ -92,13 +100,36 @@ If you press the play button, and the paragraph is fully generated, you can also
...
 
 Once your Project is converted, you have several download options available. 
 
-<img height="200" src="/product/projects/images/projects_zip_dropdown.webp" />
+<Frame>
+  <img height="200" src="/product/projects/images/projects_zip_dropdown.webp" />
+</Frame>
+
+## Auto-Regenerate
+When you have auto-regenerate enabled for your project, we will automatically check the output for any mispronunciations or unwanted audio artefacts.  If we detect any, we will automatically regenerate the audio up to twice, at no extra cost.
```
Comment (louisjoecodes) [inline product/projects/overview.mdx:116]:
```suggestion
When the Auto-Regenerate feature is enabled for your project, we will automatically check the output for any mispronunciations or unwanted audio artefacts.  If we detect any, we will automatically regenerate the audio up to twice, at no extra cost.  
```

**Review by louisjoecodes — COMMENTED** [review]
minor grammar

File: product/projects/overview.mdx [hunk 2]
```diff
@@ -15,7 +17,9 @@ Once you enter the new tab, you will encounter a screen where you can create new
 
 Click "Add a new project” and you will be presented with a popup. Here, you can choose to create a new empty project, import an already existing EPUB, PDF, TXT or HTML file, which will then automatically be converted into a project, or import text directly from a website using the URL to have the page be converted into a project. You can then use our [Audio Native](/product/audio-native/overview) feature to easily and effortlessly embed any narration project onto your website.
```
Comment (louisjoecodes) [inline product/projects/overview.mdx:18]: nit: For step-by-step workflows I would suggest using Mintlfiy [Steps](https://mintlify.com/docs/content/components/steps#steps)

**Review by louisjoecodes — APPROVED** [review]

**Review by louisjoecodes — CHANGES_REQUESTED** [review]
Minor feedback

File: product/projects/overview.mdx (hunk 2 above)
Comment (elevenlabs-laura) [reply product/projects/overview.mdx]: I will add Steps to this section on a new branch to keep things clean!

**Review by louisjoecodes — APPROVED** [review]

**Pull request merged**